docs/topics/test-system.md

```diff
@@ -1,72 +1,126 @@
 
-## Configuration
+You can find all code for running test test systems in the `/opt/test-systems` directory.
+This directory is owned by the `mediawiki` user.
+Each test system is a copy of the `example` docker-compose setup, with customized env vars and settings.
 
-The test-system is currently using a non versioned copy of the [example](../../example/README.md) folder deployed to `wb-product-testing.wikidata-dev.eqiad.wmflabs`.
+Two test systems are maintained, one for the current "latest" major release, and one for the "previous" major release.
 
-Files are located under the `/opt/` folder and for keeping some kind of papertrail a copy of the version to be tested is usually created as `/opt/wmde.1`, `/opt/wmde.2` etc.
+- Latest release (internal ports 83**)
+  - https://wikibase-product-testing.wmcloud.org (8380 internal)
+  - https://wikibase-query-testing.wmcloud.org (8381 internal)
+  - https://wikibase-qs-testing.wmcloud.org (8382 internal)
+- Previous release (internal ports 82**)
+  - https://wikibase-product-testing-previous.wmcloud.org (8280 internal)
+  - https://wikibase-query-testing-previous.wmcloud.org (8281 internal)
+  - https://wikibase-qs-testing-previous.wmcloud.org (8282 internal)
 
-```
-someone@wb-product-testing:/opt/wmde.2$ tree
-.
-├── example-fedprops
-│   ├── docker-compose.extra.yml
-│   ├── docker-compose.yml
-│   ├── extra-install.sh
-│   ├── local.env
-│   ├── LocalSettings.php
-│   ├── README.md
-│   └── template.env
-└── example-full
-    ├── docker-compose.extra.yml
-    ├── docker-compose.yml
-    ├── extra-install.sh
-    ├── LocalSettings.php
-    ├── README.md
-    └── template.env
-
-2 directories, 13 files
+These proxies are configured in Horizon https://horizon.wikimedia.org/project/proxy/
 
-```
+All of the internal ports need to be opened in the firewall for traffic from the proxies to make it to the docker hosted ports https://horizon.wikimedia.org/project/security_groups/3d459ac5-e78a-4a5e-a5e0-d31176ed3628/
 
-## Update with new release
+## Initial creation
 
+On an empty machine first you need some dependencies:
 
-Normally `example-full` is the environment you want to use. The folder should contain an `.env` file that can be updated with the new image versions produced by the [release-pipeline](https://github.com/orgs/wmde/packages?repo_name=wikibase-release-pipeline).
+- Follow the `docker` install docs at https://docs.docker.com/engine/install/debian/
+- Follow the `docker-compose` install docs at https://docs.docker.com/compose/install/
 
-### 1. stop the containers 
+You can create the directory to store the test systems:
 
 ```sh
-docker-compose -f docker-compose.yml -f docker-compose.extra.yml down
+sudo mkdir /opt/test-systems
+sudo chown mediawiki:wikidev /opt/test-systems
+sudo chmod +775 /opt/test-systems
 ```
 
-### 2. Update image version in .env file
+You can create a test system using the parameterized bash code below.
```

Comment:
Should we just create a copy of this script somewhere and say that this is the shared version we all use? Or have a versioned copy in this repo we can link to wget or curl download for example?